src/components/CountryPicker/CountrySelectorModal.js

```diff
@@ -33,13 +33,16 @@ const defaultProps = {
     onCountrySelected: () => {},
 };
 
-function filterOptions(searchValue, data) {
-    const trimmedSearchValue = searchValue.trim();
-    if (trimmedSearchValue.length === 0) {
+function searchOptions(searchValue, data) {
```

Comment:
Are `searchOptions` identical for CountrySelectorModal and StateSelectorModal? If so, we could move this function to `libs/CountrySelectorUtils`